website/docs/source/docs/boxes/base_macos.html.md

```diff
@@ -0,0 +1,160 @@
+---
+page_title: "Creating a macOS Base Box Manually"
+sidebar_current: "boxes-macos-base"
+---
+
+# Creating a macOS Base Box Manually
+
+Prior to reading this page, please check out the [basics of the Vagrant
+box file format](https://www.vagrantup.com/docs/boxes/format.html).
+
+## Pre-requisites
+
+* Install Vagrant, you can find the instructions [here](/vagrant-parallels/docs/installation)
+
+* Start Parallels Desktop and create a new macOS virtual machine as outlined  in [KB 125561](http://kb.parallels.com/125561/).
+    <div class="alert alert-info">
+    <p>
+        While creating the VM, give the username as ‘vagrant’ only.<br />
+        If this is a public box you should set the password to ‘vagrant’ as well.
+    </p>
+    </div>
+
+  * Enable passwordless sudo for macOS virtual machine
+  * Open Terminal in macOS virtual machine and run:
+
+     ```bash
+     $ sudo visudo
+     ```
+
+    Edit the line:
+
+    ```bash
+    %admin ALL=(ALL) ALL
+    ```
+
+    To Say:
+
+    ```bash
+    %admin ALL=(ALL) NOPASSWD: ALL
+    ```
+
+    Now, you should be able to run sudo without password.
```

Comment:
**suggestion:** Let's shorten the section a bit and allow a passwordless sudo for vagrant user only, following the best practices for visudo on macOS:

```suggestion
* Enable passwordless sudo for `vagrant` user on macOS virtual machine:
  * Open Terminal in macOS virtual machine and run:

     ```bash
     $ sudo visudo /private/etc/sudoers.d/vagrant
     ```

    Paste this line there and save the file (`:wq`) :

    ```bash
    vagrant ALL = (ALL) NOPASSWD: ALL
    ```

    Now `vagrant` user is allowed to run `sudo` without a password prompt.
```

Reply:
implemented